cohortextractor/mssql_utils.py

```diff
@@ -93,6 +102,24 @@ def _ctds_connect(ctds, params):
     return ctds.connect(**params)
 
 
+def stats_msg_handler(msgstate, severity, srvname, procname, line, msgtext):
+    """
+    Log parse, compile and execution timing messages sent by the server.
+    """
+    if timing_match := SQLSERVER_TIMING_REGEX.match(msgtext.decode()):
+        logger.info(
+            "sqlserver-stats",
+            description=timing_match.group("timing_type").lower().replace(" ", "_"),
+            cpu_time_secs=int(timing_match.group("cpu_time")) / 1000,
```

Comment:
suggestion: add units to the name, so "cpu_time_s" or "cpu_time_ms" so it obvious to the reader of logs straight away

Reply:
it's already got units in the name? `cpu_time_secs` is what we'll see in logs

Reply:
doh, sorry! perfect